index.html

```diff
@@ -206,7 +240,7 @@ <h3 class="recentWorkTitle wc">
               <li class="tech twc">Ruby</li>
             </ul>
           </div>
-          <button class="mybtn spwc" type="button">See Project</button>
+          <button class="mybtn spwc projectbtn" type="button">See Project</button>
```

Comment:
- [ ] All your project's content is displayed from your HTML code, but it should be displayed using JavaScript, kindly fix ix, this is one of the mandatory requirements.

As you can read below:
8. The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript. Check an example of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.

You currently have the variables created using JavaScript, but you should use this code from JavaScript instead of using HTML code to create all your projects section, to make it dinamically.

Reply:
done